view/frontend/web/template/payment/offsite-installment-form.html

```diff
@@ -0,0 +1,184 @@
+        <label data-bind="attr: {'for': getCode()}" class="label">
+            <span data-bind="text: getTitle()"></span>
+        </label>
+        <div data-bind="visible: !isActive()" class="page messages">
+            <div role="alert" class="messages">
+                <div class="message-warning warning message">
+                    <div data-bind="i18n: 'Option available only for orders in THB'"></div>
+                </div>
+            </div>
+        </div>
+        <div data-bind="visible: orderValueTooLow()" class="page messages">
+            <div role="alert" class="messages">
+                <div class="message-warning warning message">
+                    <div data-bind="i18n: 'Minimum order value is 5000 THB'"></div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="payment-method-content">
+        <!-- ko foreach: getRegion('messages') -->
+        <!-- ko template: getTemplate() -->
+        <!-- /ko -->
+        <!--/ko-->
+        <div class="payment-method-billing-address">
+            <!-- ko foreach: $parent.getRegion(getBillingAddressFormName()) -->
+            <!-- ko template: getTemplate() -->
+            <!-- /ko -->
+            <!--/ko-->
+        </div>
+        <div class="omise-banking-fee-warn" data-bind="i18n: 'Choose your installment terms'"></div>
+        <form class="form" data-bind="attr: {
+              id: getCode() + 'Form',
+              }">
+
+            <fieldset data-bind="attr: {class: 'fieldset payment items ' + getCode(), id: 'payment_form_' + getCode()}">
+                <ul class="form-list">
+                    <!-- KTC -->
+                    <li class="items">
+                        <input type="radio" name="offsite" value="installment_ktc" data-bind="
+                            attr: {
+                                id: getCode() + '_ktc'
+                            },
+                            checked: omiseOffsite,
+                            click: function(data, event) { 
+                                    installmentTerms(null); 
+                                    return true; 
+                                }" />
+                        <label for="omise_offsite_installment_ktc">
+                            <div class="omise-logo-wrapper ktc">
+                                <img class="ktc" />
+                            </div>
+                            <div class="omise-banking-text-wrapper">
+                                <span class="title" data-bind="i18n: 'Krungthai Card'"></span>
+                            </div>
+                        </label>
+                        <select style="margin-top:10px" data-bind="options: getInstallmentTerms('ktc'),
+                                        optionsText: function(item) {
+                                            return item;
+                                        },
+                                        valueAllowUnset: true,
```

Comment:
Seems like it just displays a list of numbers? We should really be telling the user somewhere that this is the number of monthly payments. A number alone probably is not enough

Reply:
it will be subject for future PR

Reply:
I think it needs to go in now. The number alone is essentially meaningless to the customer

Reply:
@jacstn in case if you missed this one